### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by bakura10**
Hi!

This PR only contains some very minor cleanup to match ZF codestyle and small PHPDoc fixes.

### CODE REVIEW

File: src/AbstractSerializer.php [hunk 1]
```diff
@@ -38,7 +38,7 @@ protected static function getLine(StreamInterface $stream)
     {
         $line    = '';
         $crFound = false;
-        while (! $stream->eof()) {
+        while (!$stream->eof()) {
```
Comment (weierophinney) [inline src/AbstractSerializer.php]: Honestly, this is a personal preference thing, and I've actually been changing these in the ZF2 code base to `! $...` when I come across them. Let's leave these as-is, please.

File: src/Request.php [hunk 2]
```diff
@@ -40,9 +40,7 @@ public function __construct($uri = null, $method = null, $body = 'php://memory',
     public function getHeaders()
     {
         $headers = $this->headers;
-        if (! $this->hasHeader('host')
-            && ($this->uri && $this->uri->getHost())
-        ) {
+        if (!$this->hasHeader('host') && ($this->uri && $this->uri->getHost())) {
```
Comment (weierophinney) [inline src/Request.php]: This was easier to read as two lines.

File: src/Request.php [hunk 3]
```diff
@@ -54,10 +52,8 @@ public function getHeaders()
      */
     public function getHeader($header)
     {
-        if (! $this->hasHeader($header)) {
-            if (strtolower($header) === 'host'
-                && ($this->uri && $this->uri->getHost())
-            ) {
+        if (!$this->hasHeader($header)) {
+            if (strtolower($header) === 'host' && ($this->uri && $this->uri->getHost())) {
```
Comment (weierophinney) [inline src/Request.php]: This was easier to read as two lines.

File: src/ServerRequestFactory.php [hunk 4]
```diff
@@ -227,7 +226,6 @@ public static function marshalHeaders(array $server)
      * @param array $server
      * @param MessageInterface $request
      * @return Uri
-     * @deprecated as of 0.7.0; use marshalUriFromServer() instead.
```
Comment (weierophinney) [inline src/ServerRequestFactory.php]: We should remove the method, really.

Comment (Ocramius) [inline src/ServerRequestFactory.php]: 2.0.0 :-\

File: src/UploadedFile.php [hunk 5]
```diff
@@ -60,40 +60,42 @@ public function __construct($streamOrFile, $size, $errorStatus, $clientFilename
...
             throw new InvalidArgumentException('Invalid size provided for UploadedFile; must be an int');
         }
+
         $this->size = $size;
 
-        if (! is_int($errorStatus)
-            || 0 > $errorStatus
-            || 8 < $errorStatus
-        ) {
+        if (!is_int($errorStatus) || 0 > $errorStatus || 8 < $errorStatus) {
```
Comment (weierophinney) [inline src/UploadedFile.php]: I prefer multiple lines here, as it's far simpler to see and understand the specific conditions than when on the same line.

File: src/Uri.php [hunk 6]
```diff
@@ -275,7 +275,7 @@ public function withHost($host)
      */
     public function withPort($port)
     {
-        if (! (is_integer($port) || (is_string($port) && is_numeric($port)))) {
+        if (!is_numeric($port)) {
```
Comment (weierophinney) [inline src/Uri.php]: I could have sworn `is_numeric()` did not return `true` for non-string values, but I just tested, and it does. Good change.

File: src/ServerRequestFactory.php (hunk 4 above)
Comment (weierophinney) [inline src/ServerRequestFactory.php]: I should have removed it before creating the 1.0.0 tag. I messed up. :-(

Comment (bakura10) [inline src/ServerRequestFactory.php]: Okay, I’m too lazy to revert all the space changes, maybe it’s easier for you to pick the two-three interesting changes?

Too bad to have deprecated methods in 1.0.0… :( BUT you can tag 2.0.0 as that’s the main advantage of splitting repositories (or let’s remove them and add a BC notice… this thing has been released yesterday, I hope nobody already use that in production and using deprecated features)

Comment (weierophinney) [inline src/ServerRequestFactory.php]: Sure, I'll cherry-pick when I review for merge.

File: src/ServerRequestFactory.php [hunk 7]
```diff
@@ -464,9 +468,8 @@ private static function createUploadedFileFromSpec(array $value)
      * @param array $files
      * @return UploadedFileInterface[]
      */
-    private static function normalizeNestedFileSpec(array $files)
+    private static function normalizeNestedFileSpec(array $files = [])
     {
-        $files = [];
```
Comment (weierophinney) [inline src/ServerRequestFactory.php]: Actually, I think you've spotted an error; the value shouldn't be reset at the start of the method!

**Pull request closed without merging**